src/30_Advanced/Scrollbound_Effects_and_Animations.html

```diff
@@ -0,0 +1,353 @@
+     */
+    .carousel-container {
+      margin-right: -60%;
+      margin-left: 60%;
+    }
+  </style>
+</head>
+
+<body>
+
+  <!--
+  ## Scrollbound Animations
+
+  Let's create a fidget spinner that rotates as user scrolls the page.
+
+  This sample showcases the core concept behind combining
+  `amp-position-observer` and `amp-animation`: The ability to progress
+  through a keyframe animation timeline as an element progresses through the viewport via scrolling.
+  -->
+
+  <!--
+  We define our fidget spinner scene as a `div` with an `image`.
+
+  We add an `amp-position-observer` element as child of the scene to monitor
+  its progress through the viewport (position observer acts on its immediate parent by default).
+
+  By defining an `on:scroll` action, we seek through the timeline of the keyframe
+  animation that we will define later in the next section.
+
+  By setting `intersection-ratios`, we define how much of the scene should be
+  visible in the viewport before `amp-position-observer` triggers any of its events.
+
+  Here, we like the spinning to happen only when fidget spinner is fully visible so
+  we set it to `1`.
+  -->
+  <div class="fidget-spinner-scene">
+
+    <amp-position-observer on="scroll:spinAnimation.seekTo(percent=event.percent)" intersection-ratios="1" layout="nodisplay">
+    </amp-position-observer>
+
+    <amp-img id="fidgetSpinnerImage" width="1024" height="1114" layout="responsive" src="/img/fidget.png" alt="Picture of a fidget spinner"></amp-img>
+  </div>
+
+  <!--
+   Now we need to define the animation itself:
+
+   Fairly straightforward in this case, we want `fidgetSpinnerImage` to rotate
+   360 degrees, so we just define a `"transform": "rotate(360deg)"` as the last (and only)
+   keyframe.
+
+   The value of `duration` is irrelevant in this case since we progress through
+   the timeline via scoling so we just set it to `1`.
+  -->
+  <amp-animation id="spinAnimation" layout="nodisplay">
+    <script type="application/json">
+      {
+        "duration": "1",
+        "fill": "both",
+        "direction": "reverse",
+        "animations": [
+          {
+            "selector": "#fidgetSpinnerImage",
+            "keyframes": [
+              { "transform": "rotate(360deg)" }
+            ]
+          }
+        ]
+      }
+    </script>
+  </amp-animation>
+
+  <!--
+  ## Fade & Slide Transitions
+
+  Now that we have learned the basic core concepts behind `amp-position-observer`
+  and `amp-animation`, let's dive into more creative ways they can be combined to
+  create interesting transitions.
+
+  In this sample, we will combine timebound and scrollbound transitions together
+  to create an effect where the opacity of the card is tied to how much of it
+  is visible in the viewport (scrollbound) and the title of the card animates in/out
+  (timebound) as the card enters and exits the viewport.
+  -->
+
+  <!--
+  Our card scene is simply composed of an image and an overlayed title.
```

Comment:
Same suggestion as before, paragraph or bullets.

Reply:
done, restructured and added more details as bullet points.